about/licenses/index.md

```diff
@@ -0,0 +1,85 @@
+---
+title: LLNL Software Licensing
+layout: default
+---
+
+## {{ page.title }}
+{: .page-header}
+
+*This page contains information meant for LLNL Employees with instructions for
+how to mark their source code projects. It is not intended to serve as legal
+advice*
+
+### Required Content
+
+All LLNL software must contain the following markings:
+
+#### 1. README file
+
+All projects shall have a `README.md` file at the root of the repository,
+including at least the following information:
+
+- "Getting Started" instructions
```

Comment:
I would argue that the "getting started" instructions are just a nice-to-have. The bare minimum *requirement* is a "Releasability" Section.

Reply:
I'm fine with that.